webapp/src/locale/po/README.md

```diff
@@ -0,0 +1,10 @@
+# Legacy locales
+
+To add a new locale, make sure to add/edit the following files:
+
+- `legacy/application/models/Locale.php`
+- `legacy/locale/<LANG>/LC_MESSAGES/libretime.po`
+- `legacy/public/js/datatables/i18n/<LANG>.txt`
+- `legacy/public/js/plupload/i18n/<LANG>.js`
+
+The `legacy/application/controllers/LocaleController.php` contains additional translations loaded by jquery i18n `$.i18n` and used with `$.i18n._`.
```

Comment:
Please remove any duplicate or unneeded files.

All the PO files can be removed, the JSON files can be cleaned to only include the keys we actually use in the code.